use_sim_time: true
verbose: false
world_frame: "world"
color_mode: "color"

method: "fast"
sensor_horizontal_resolution: 1280
sensor_vertical_resolution: 960
sensor_vertical_field_of_view_degrees: 360
integrator_threads: 8

tsdf_voxel_size: 0.05
truncation_distance: 0.15
# max_weight: 1000
pointcloud_queue_size: 10
max_ray_length_m: 6 # 14 or 16m would be better, but for that we need better odometry. Otherwise the surface get too damaged.
min_ray_length_m: 0.3
allow_clear: true
# max_consecutive_ray_collisions: 10
use_const_weight: true
# use_weight_dropoff: false
voxel_carving_enabled: true
# use_sparsity_compensation_factor: true
# sparsity_compensation_factor: 20.0

# publish_pointclouds_on_update: true
# slice_level: 1.0
# publish_slices: true
# update_mesh_every_n_sec: 1.0

pointcloud_deintegration_queue_length: 0

publish_tsdf_map: true
publish_esdf_map: true
publish_map_every_n_sec: 0
publish_map_with_trajectory: true

submap_interval: 5

